spec.md

```diff
@@ -570,13 +559,297 @@ schema being used.
         "bullets": [
           "40 concurrent connections"
         ]
+      },
+      "schemas": {
+        "service_instance": {
+          "create": {
+            "parameters": {
+              "$ref": "osb_v2:///fakeservice/v1#BillingAccountParameters"
+            }
+          },
+          "update": {
+            "parameters": {
+              "$schema": "http://json-schema.org/draft-04/schema#",
+              "type": "object",
+              "propeties": {
+                "some-other-account": {
+                  "description": "Other fake account needed for update.",
+                  "type": "string"
+                },
+                "billing": {
+                  "$ref": "osb_v2:///fakeservice/v1#BillingAccountParameters"
+                }
+              }
+            }
+          }
+        },
+        "service_binding": {
+          "create": {
+            "parameters": {
+              "$ref": "osb_v2:///fakeservice/v1/bindings#AccountParameters"
+            }
+          }
+        }
       }
     }]
   }]
 }
 ```
 
 
+## JSON Schemas
+
+The JSON Schemas endpoint is used to define reusable definitions of JSON Schema
+Objects referenced throughout the [Catalog](#catalog-management). If a Platform
+intends to support JSON Schemas, it MUST examine the response from the catalog
+to determine if the Service Broker is using `$ref` JSON Schemas. The Service
+Broker can signal to the Platform that JSON Schema components are supported by
+using the `osb_v2:` scheme. Components are a way to allow Platforms to only
```

Comment:
is the osb_v2 schema necessary to satisfy the requirements? 
I wonder if it is equivalent to http:///v2/catalog/schemas/

(basically a relative link).

Reply:
that is technically not a relative link, it is a authority-less URL and I don't think it is enough to make assumptions on if we can capture the path and use it for the component id. The osb_v2 is the signal for a component path.

If we just went off `http:///` we might hijack someones valid implementation.

Reply:
can you please elaborate on the hijacking issue?

Reply:
there is nothing that prevents an author of json schema to make their document `$id` to be `http:///something/` with sub-documents be appended. So if we redirect `http:///` to mean schema fetch from broker, then we might be smashing someones implementation. TBH there is nothing that prevents existing impls to use `osb_v2:///` today, it would just be a really strange choice for them.